### ISSUE TITLE
[Bug] Client freeze with quest timer

### ISSUE THREAD

**Issue by kevlhw** [ElvUI-TBC/ElvUI#301]
Client freezes/crashes when quest timer is displayed. I believe it's an ElvUI error because disabling only ElvUI fixes the issue.

Steps to reproduce:

1. Have latest release of ElvUI
2. Accept quest that shows a Quest Timer
3. Client freezes or crashes
4. Restart client, disable ElvUI
5. Reenter game, game does not freeze.
6. Reenable ElvUI, game freezes again

Other addons:
Atlas + Atlasloot
Omen + Omen Threat
Zygor Guide

Thanks for porting & supporting this great addon!

**Comment by crumdidlyumshis** [comment]
Im not sure what could be the cause of this, but I did exactly the steps you provided with the same addons. No crash here.

Is this a consistent crash? If so, could you provide a video or any sort of logs that might show a reason as to why this is happening?

**Comment by KevinTyrrell** [comment]
Had this occur to me last night and today. First quest was [20] The Sleeper Has Awakened. I accepted the quest, exactly three seconds passed, then a full screen freeze in which my client did not recover from (had to alt-f4). Today I went to Azuremyst Isle, accepted a timed quest where it automatically mounted me on an 100% mount and the quest had a timer. My client also freezed within 3 seconds.



When I log back in, within 1-2 seconds the client completely freezes (as long as the quest timer is displayed).



I disabled all of my other addons except for ElvUI and no freeze occurs.



I tested each and every addon I had and finally got to one named "QuestHelper". Version 0.50 [located here](https://wow.curseforge.com/projects/quest-helper/files/88684). Indeed this addon freezes your client only moments after a quest with a timer runs.


On it's wowwiki page it shows [that it has a tendency to cause client freezes](https://wowwiki.fandom.com/wiki/QuestHelper).



I'm fairly sure this is what happened with @kevlhw .
